server/core/src/https/views/reset.rs

```diff
@@ -600,7 +617,7 @@ pub(crate) async fn view_self_reset_get(
             client_auth_info,
             kopid,
             jar,
-            "/ui/update_credentials",
+            "/ui/update-credentials",
```

Comment:
As per previous comment

```suggestion
            "/ui/update_credentials",
```